parsl/monitoring/db_manager.py

```diff
@@ -15,7 +15,7 @@
 from parsl.process_loggers import wrap_with_logs
 from parsl.utils import setproctitle
 
-logger = logging.getLogger("database_manager")
+logger = logging.getLogger(__name__)
```

Comment:
there's nothing to direct this to runinfo/NNN/database_manager.log, which with this PR ends up empty - this code already used `set_file_logger` from parsl.log_utils, but directed at the previous `"database_manager"` name.

Run `pytest parsl/tests/ --config parsl/tests/configs/htex_local_alternate.py` to see what database_manager.log